modules/stanford_profile_styles/lib/scss/stanford_profile_styles.scss

```diff
@@ -40,6 +40,7 @@ svg {
 }
 
 // Hide the SVG in the lockup.
+.su-med-logo,
```

Comment:
Where does this come from?

Reply:
@sherakama,
This is from the BioE site:
https://bioedrupal8.sites.stanford.edu/ 
Because the URL points to https://med.stanford.edu/
and may encounter the same external link issue.

Reply:
Oh huh. Hacked that in there did ya. 
Because that is non-standard, please remove it from this PR. That belongs in the soe_basic theme.